docs/staking.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Staking in Oxford and later protocols
+id: staking
+author: Alireza Haghshenas
+---
+
+Since the first Tezos protocol, Tez owners could participate in baking, endorsement (now renamed to attestation), and voting by `delegating` their tez to bakers.
+Starting with Oxford, there is now an additional way of participating which is called `staking`. You can read more about it (here)[https://tezos.gitlab.io/protocols/018_oxford.html#adaptive-issuance-ongoing] and (here)[https://spotlight.tezos.com/announcing-oxford-tezos-15th-protocol-upgrade-proposal/] (under "Adaptive Issuance and Staking").
+
+In Taquito, we support three new operations: `stake`, `unstake`, and `finalize_unstake`.
```

Comment:
probably better to specify that they are pseudo operations to not cause confusion